src/app/georeferencer/qgsgeorefmainwindow.cpp

```diff
@@ -1181,20 +1344,57 @@ void QgsGeoreferencerMainWindow::addRaster( const QString &file )
   QgsRasterLayer::LayerOptions options;
   // never prompt for a crs selection for the input raster!
   options.skipCrsValidation = true;
-  mLayer = std::make_unique< QgsRasterLayer >( file, QStringLiteral( "Raster" ), QStringLiteral( "gdal" ), options );
+
+  mRLayer = std::make_unique< QgsRasterLayer >( file, QStringLiteral( "Raster" ), QStringLiteral( "gdal" ), options );
+
+  // so layer is not added to legend
```

Comment:
This is reverting a recent change which avoided the layer being added to the project -- can you revert this reversion please?

Reply:
I thought this was the change that was implemented as previously it was a unique ptr and it didn't fix itself when I rebased. Isn't that the same as https://github.com/qgis/QGIS/blob/c6370c7f52d561f282ff084a6e1cb136a66aa5f2/src/app/georeferencer/qgsgeorefmainwindow.cpp#L1185 ??

Reply:
Removed the line below though